src/routes/index.svelte

```diff
@@ -37,12 +37,47 @@
 	async function updateEntries() {
 		fetchIndex += 1
 		let thisFetchIndex = fetchIndex
-		const res = await fetch(
-			`/api/entries.json?visible=${showVisible}&unlisted=${showUnlisted}&hidden=${showHidden}`
-		)
+		let search_value = (document.getElementById('search') as HTMLInputElement).value || null
+
+		let tags: string
+		let query: string
+		if (!search_value) {
+			query = undefined
+		} else {
+			let all = search_value.split(' ')
+			query = all.filter(word => !word.startsWith('tags:')).join(' ')
+			if (search_value.includes('tags:')) {
+				let tags_raw = all.filter(word => word.startsWith('tags:'))[0]
+				tags = tags_raw.slice(5).replaceAll('_', '%20')
+				console.log(query)
+			}
+		}
+
+		let url = `/api/entries.json?visible=${showVisible}&unlisted=${showUnlisted}&hidden=${showHidden}`
+
+		if (query) {
+			url += `&query=${query}`
+		} else if (tags) {
```

Comment:
Fix so you can search with both a query and tags
```suggestion
		}
		if (tags) {
```